src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts

```diff
@@ -232,10 +234,24 @@ export class MarkdownToolbarComponent extends AngularDisposable {
 			if (this.cellModel.currentMode !== CellEditModes.WYSIWYG) {
 				needsTransform = false;
 			} else {
+				let result = undefined;
+				const isFile = URI.parse(linkCalloutResult?.insertUnescapedLinkUrl).scheme.includes('file');
```

Comment:
Is there a reason you're using includes here instead of ===? Do you expect other schemes containing file to be valid?

Reply:
For now, we only expect file schemes